go/src/koding/kites/config/konfig.go

```diff
@@ -170,6 +170,7 @@ type Konfig struct {
 	PublicBucketRegion string `json:"publicBucketRegion,omitempty"`
 
 	LockTimeout time.Duration `json:"lockTimeout,omitempty"`
+	NoMetrics   bool          `json:"noMetrics,string,omitempty"`
```

Comment:
I'm not looking from Go developer perspective but rather from user one. `set noMetrics true` is less readable than `set metrics false`. Just a personal opinion. If you think this fits better, let it be.